src/tests/suite/daos_test_common.c

```diff
@@ -1070,25 +1073,29 @@ get_server_config(char *host, char *server_config_file)
 			break;
 	}
 
-	pch = strtok(line, " ");
-	while (pch != NULL) {
-		if (strstr(pch, "--config") != NULL) {
-			if (strchr(pch, '=') != NULL)
-				strcpy(server_config_file,
-				       strchr(pch, '=') + 1);
-			else {
+	if (conf)
+		strcpy(server_config_file, DAOS_SERVRE_CONF);
+	else {
+		pch = strtok(line, " ");
+		while (pch != NULL) {
+			if (strstr(pch, "--config") != NULL) {
+				if (strchr(pch, '=') != NULL)
+					strcpy(server_config_file,
+				       	strchr(pch, '=') + 1);
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs